docs/pages/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -199,8 +200,58 @@ version: v2
 ```
 
 Now the alice user can only access `env:production` apps using a trusted device.
-For example, to access an app called `myapp`, alice runs `tsh proxy app myapp -p
-8888` and opens http://localhost:8888 in her browser.
+
+## Desktop Access support
+
+Desktop access may enforce device trust via [role-based enforcement](
```

Comment:
Does the Teleport Desktop Service perform the enforcement?

Reply:
Yes, all services will look at your certificate extensions if enforcement is on.

Reply:
👍 I think it would make more sense here to say "The Teleport Desktop Service may enforce...".

Reply:
f2250a9945 - done for both Desktop and App sections.